e2e/cypress/tests/support/api/bots.ts

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {Bot} from 'mattermost-redux/types/bots';
+
 import {getRandomId} from '../../utils';
 
 // *****************************************************************************
 // Bots
 // https://api.mattermost.com/#tag/bots
 // *****************************************************************************
 
-Cypress.Commands.add('apiCreateBot', ({prefix, bot = createBotPatch(prefix)} = {}) => {
+function apiCreateBot({prefix, bot = createBotPatch(prefix)}): Cypress.Chainable<{bot: Bot & {fullDisplayName: string}}> {
```

Comment:
We're no longer using types from `mattermost-redux`. Instead, you may use directly from the Cypress namespace where types are imported in one place ([here](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/tests/support/index.d.ts#L7-L35)).
```suggestion
function apiCreateBot({prefix, bot = createBotPatch(prefix)}): Cypress.Chainable<{bot: Cypress.Bot & {fullDisplayName: string}}> 
```